user_guide/data_management_grids.rst

```diff
@@ -250,8 +250,8 @@ If permitted, you can delete records.
 
 |
 
-In the same manner, you can merge several records (if applicable). Currently, merge can only be done for 
-:ref:`accounts <user-guide-accounts>`.
+In the same manner, you can merge several records (if corresponding :ref:`capabiltiy <admin-capabilities-merge>` has 
+been enabled). Currently, merge can only be done for :ref:`accounts <user-guide-accounts>`.
```

Comment:
Edit: In the same manner, you can merge several records (if the corresponding :ref:`capability <admin-capabilities-merge>` has been enabled). Currently, merge can only be done for :ref:`accounts <user-guide-accounts>`.